src/hooks/useJournalData.ts

```diff
@@ -216,7 +219,7 @@ function isJournalRecord(val: any): val is JournalRecord {
 const useJournalData = (
     journalName?: string,
     desiredCount: number = 50,
-    maxBytes: number = 5 * 10 ** 6
+    maxBytes: number = 16 * 10 ** 6
```

Comment:
Can you add your PR comment of `16mb, which is the max document size, ensuring we'll always get at least 1 doc if it exists` here as a code comment so we don't lose why this is 16 in the codebase.